demos/sourceDataProcessing/getOptionSource.m

```diff
@@ -0,0 +1,51 @@
+% (C) Copyright 2019 CPP BIDS SPM-pipeline developers
+
+function optSource = getOptionSource()
+  %
+  % Returns a structure that contains the options chosen by the user to run the source processing
+  % batch workflow
+  %
+  % USAGE::
+  %
+  %   optSource = getOptionSource()
+  %
+  % :returns: - :optSource: (struct)
+
+  if nargin < 1
+    optSource = [];
+  end
+
+  % Set the folder where sequences folders exist
```

Comment:
Something we have to decide regarding documentation is whether we want to put the comments that describe the structure fields directly in the help section.

Reply:
mmh, yes it could be definitely usefule. An hackadoc is maybe required for consistency :)